src/EditorFeatures/CSharpTest/EditorConfigSettings/Updater/SettingsUpdaterTests.cs

```diff
@@ -17,25 +21,37 @@
 using Microsoft.CodeAnalysis.Test.Utilities;
 using Microsoft.CodeAnalysis.Text;
 using Roslyn.Test.Utilities;
+using Roslyn.Utilities;
 using Xunit;
 
 namespace Microsoft.CodeAnalysis.Editor.UnitTests
 {
     [UseExportProvider]
     public partial class SettingsUpdaterTests : TestBase
     {
-        private static Workspace CreateWorkspaceWithProjectAndDocuments()
+        private static async Task<Workspace> CreateWorkspaceWithProjectAndDocumentsAsync()
```

Comment:
I don't see how this test file is C#-specific. It looks relevant to both languages. If VB support won't come in this PR, this can be done later.

Reply:
yep adding as part of #52696